excellaco/excella-feedback
# Deployment Prep

- Changed seeds file so that there is separate seed data between dev & prod environments 
- Added Figaro gem to store production environment variables 

To test this PR: 
- Run `$ rails db:create:all` to create all databases for all environments 
- Run `$ rails db:migrate RAILS_ENV=production` to run migration on prod database
- Run `$ rails db:seed RAILS_ENV=production` to run production seeds file 
- Run `$ rails s -e production` to see app in production environment 
- (Optional, be careful) If necessary, run `$ rails db:reset RAILS_ENV=production DISABLE_DATABASE_ENVIRONMENT_CHECK=1` to reset database, run production migration, & run production seeding
